dist/images/ovnkube.sh

```diff
@@ -180,6 +175,11 @@ check_ovn_daemonset_version () {
 # (on non-master call oc get ep ...)
 get_master_ovn_vars () {
   master_host=$(hostname)
+
+  ovn_master_host=$(grep -m 1 -w ${master_host} /etc/hosts | awk '{ print $1 }')
+  if [[ ${ovn_master_host} == "" ]] ; then
+    ovn_master_host=$(host ${master_host} | awk '{ print $4 }')
```

Comment:
Could we use 'getent ahosts' here instead of 'host'? Not everywhere will have the bind utils, but I'd hope everywhere has glibc.

Reply:
@dcbw I agree with you here. Note that `host` was in use in the current code, I just enabled it to work in Ubuntu-based image. I am going to change it to use getent. Thanks.